modules/Preprocessing.py

```diff
@@ -0,0 +1,115 @@
+from datetime import datetime
+import re
+import string
+from typing import List, Dict
+from nltk.stem.wordnet import WordNetLemmatizer
+from nltk.corpus import stopwords
+from nltk.tag import pos_tag
+from nltk.tokenize import word_tokenize
+
+
+# lazy load stopwords
+_stopwords = stopwords.words('english')
+_stopwords.extend(['twitter', 'nt'])
+
+
+def _handle_date(date_string: str):
+    """
+    We receive dates from tweets in the format 'yyyy-m-d'.
+    This function returns a datetime object with proper formatting (yyyy-mm-dd)
+    """
+    # return date(*[int(date) for date in date_string.split(',')]) # sorry we didnt get to use you ;(
+    return datetime.strptime(date_string, '%Y,%m,%d').date()
```

Comment:
I'm not sure I understand why this i necessary

Reply:
Reading it from a file, as I initially was told to do, dates were just strings. As a part of preprocessing, I figured it'd make sense to have actual datetime objects for presentation, rather than just strings.

If we are no longer loading from files _at all_, then it should be removed.

Reply:
Ah okay. The dates will still be strings as part of the strings containing the twitter objects so it makes sense to keep the function